semigroupoids.cabal

```
@@ -54,7 +54,7 @@ description:
   indicate a subclass relationship that actually exists; dashed edges indicate a
   subclass relationship that /should/ exist, but currently doesn't.
   .
-  <<img/classes.svg Relationships among type classes from this package and others>>
+  <<docs/img/classes.svg Relationships among type classes from this package and others>>
```

What is this `docs/` directory? Is this a documented requirement somewhere that files like this have to live in a `docs`/ directory?

---

Hm. It seems a bit fragile to assume what directory structure Haddock will use, as that is a detail that Hackage could change in the future. I'm not hugely fond of symlinking either, as it has portability issues on Windows.

I think I would prefer using an absolute path to the location on GitHub where the image lives. It's not perfect, but at least it will work consistently on Hackage and in local Haddocks. Something like this:

```suggestion
  <<https://raw.githubusercontent.com/ekmett/semigroupoids/master/img/classes.svg Relationships among type classes from this package and others>>
```

This is what we're doing in the `README` already, so there's some precedent for this.

---

Works for me; done.